src/index.js

```diff
@@ -109,6 +109,7 @@ function loadConfig() {
 
     // use a unique container for assets for each PR inside the shared storage account
     config.assetsContainerName = `${config.assetsContainerName}-${config.pullRequestId}`;
+    config.isPullRequest = true;

```

Comment:
`config.isPullRequest` is already `true` here

Reply:
I was using your old repo code locally which didn't have this in the config :) Will fix this.